airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -87,7 +87,7 @@ def Popen(self, cmd, **kwargs):
         :param kwargs: extra arguments to Popen (see subprocess.Popen)
         :return: handle to subprocess
         """
-        self.log.info("Executing command: %s", ' '.join(cmd))
```

Comment:
The function to mask the password was created but never called on. We should mask it in the `cmd` when we're logging it.